CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ Looking for changes that affect our C API? See the [C API Changelog](lib/c-api/C
 ## **[Unreleased]**
 
 ### Changed
+- [#2478](https://github.com/wasmerio/wasmer/pull/2478) Rename `traps` input to `wasm_instance_new()` to `trap`.
```

Comment:
This is incorrect, it should have been written in the `lib/c-api/CHANGELOG.md` file.

Reply:
Fixed in https://github.com/wasmerio/wasmer/pull/2481.

Reply:
Sorry! I noticed that only after committing. Thank you for fixing it.